File: IDAUtil/Model/Properties/TcodeProperty/VA02Obj/OrderObjects/Rejections/RejectionsSapOrderProperty.cs
using System.Collections.Generic;

namespace IDAUtil.Model.Properties.TcodeProperty.VA02 {
    public class RejectionsSapOrderProperty {
        public string salesOrg { get; set; }
        public int orderNumber { get; set; }
        public List<RejectionsSapLineProperty> lineDetails { get; set; }

        public RejectionsSapOrderProperty() { }
    }

    public class RejectionsSapLineProperty {
        public int sku { get; set; }
        public int lineNumber { get; set; }
        public double orderedQty { get; set; }
        public double confirmedQty { get; set; }
        public string rejectionCode { get; set; }
        public bool isReplacePartialCut { get; set; }
        public string reason { get; set; }

        public RejectionsSapLineProperty() { }
    }
}

File: IDAUtil/Model/Properties/TcodeProperty/VA02Obj/OrderObjects/Rejections/RejectionsSapOrderProperty.vb
Public Class RejectionsSapOrderProperty
    Public Property salesOrg As String
    Public Property orderNumber As Long
    Public Property lineDetails As List(Of RejectionsSapLineProperty)

    Public Sub New()
    End Sub
End Class

Public Class RejectionsSapLineProperty
    Public Property sku As Long
    Public Property lineNumber As Long
    Public Property orderedQty As Long
    Public Property confirmedQty As Long
    Public Property rejectionCode As String
    Public Property isReplacePartialCut As Boolean
    Public Property reason As String

    Public Sub New()
    End Sub
End Class